server/blob/blob-deduplicating/src/main/scala/org/apache/james/server/blob/deduplication/DeDuplicationBlobStore.scala

```diff
@@ -29,6 +29,7 @@ import org.apache.commons.io.IOUtils
 import org.apache.james.blob.api.{BlobId, BlobStore, BucketName, DumbBlobStore}
 import org.reactivestreams.Publisher
 import reactor.core.publisher.Mono
+import reactor.core.scala.publisher.SMono
```

Comment:
Logically you should remove the `import reactor.core.publisher.Mono` above then?

Reply:
No it is used for Mono::using